build_projects/dotnet-host-build/build.ps1

```diff
@@ -17,7 +20,10 @@ if($Help)
     Write-Host ""
     Write-Host "Options:"
     Write-Host "  -Configuration <CONFIGURATION>     Build the specified Configuration (Debug or Release, default: Debug)"
-    Write-Host "  -Architecture  <ARCHITECTURE>      Build the specified architecture (x64 or x86 (supported only on Windows), default: x64)"
+    Write-Host "  -Architecture  <ARCHITECTURE>      Build on the specified architecture (x64 or x86 (supported only on Windows), default: x64)"
+    Write-Host "  -TargetArch  <ARCHITECTURE>        Build for the specified architecture (x64, x86 (supported only on Windows), or arm64, default: x64)"
+    Write-Host "  -ToolsetDir  <TOOLSETDIR>          Path to a toolset to use when building the native host. Used for ARM64. )"
```

Comment:
I would call out that `--ToolsetDir` is "temporarily supported for Arm64 only" to let folks know this will go away.